### ISSUE TITLE
La introducción al problema debe ser específica, no genérica.

### ISSUE THREAD

**Issue by JJ** [VictorMorenoJimenez/tfg2020#99]
Es decir, tu trabajo no es un trabajo que proponga DevOps, sino que usa DevOps para resolver un problema... Problema que es el que tienes que plantear, de forma concreta y demostrar que se demuestre al final que el problema está resuelto. Como se ha indicado en el issue anterior, debes ser muy concreto sobre el problema que quieres solucionar y qué harás para considerar el problema resuelto.

**Comment by VictorMorenoJimenez** [comment]
@JJ Tengo una duda con esto, si que es verdad que en el capítulo de introducción hago una introducción genérica explicando el problema con un poco de perspectiva. Sin embargo más adelante en el capítulo 2 Análisis, se concreta bastante el problema y qué se pretende resolver y se plantean las casuísticas que se puedendar y como se resuelven. 
Me gustaría saber si con este issue te refieres simplemente a la introducción del capítulo 1, o si también la definición del problema del capítulo 2 es errónea.

**Comment by JJ** [comment]
@VictorMorenoJimenez Dependerá de cómo lo hayas hecho en el uno; la definición del problema tiene que ir al principio, no debes repetirlo.

**Comment by VictorMorenoJimenez** [comment]
@JJ Sí, pero una es introducción donde se pone en contexto y la otra en la sección 2 es la definición más específica. Es por eso que me gustaría aclararlo ya que son dos cosas distintas, o por lo menos así lo he pretendido

**Comment by JJ** [comment]
No hagas esperar al capítulo 2 para saber el problema que se está resolviendo. La introducción siempre va primero, y tiene que incluir la motivación y los objetivos. Si no sabes qué problema se está resolviendo, el contexto (o estado del arte) importa poco.

**Comment by VictorMorenoJimenez** [comment]
@JJ Buenas, estoy redactando los objetivos y me surgen dudas... El objetivo del proyecto es asegurar automatizar el despliegue de la infraestructura de una empresa y después asegurar que los cauces de producción de software sean seguros y rápidos. Estos serían los dos objetivos principales. Mi pregunta es: ¿Hay que desgranar estos dos objetivos principales en objetivos más pequeños como: Asegurar cluster con firewall redundante o Garantizar contrl de versiones ?

**Comment by JJ** [comment]
@VictorMorenoJimenez Lo primero no es un objetivo, lo segundo sí. Los objetivos deben ser medibles y no simplemente descripción de tareas. En el segundo caso afirmas que debe ser seguro (y que se ha alcanzado este objetivo de seguridad tendrás que probarlo) y rápido (tendrás que probar cuanto se tarda en este cauce específico de producción). El primero, "automatización", es una tarea, no un objetivo. El objetivo es el proceso de negocio que quieres mejorar mediante la automatización, y debe ser medible. Los subobjetivos sí pueden ser tareas más concretas, pero recuerda siempre que los subobjectivos deben estar relacionados con los requisitos funcionales y por tanto con los issues.
